docs/source/examples/quantum_simulation_1d_ising.md

```diff
@@ -0,0 +1,677 @@
+
+```{code-cell}
+:id: TPkKegw0jNV-
+
+L = 5
+h_z = 0.9
+h_x = 0.5
+Jdt = 0.5
+n_dt = 6 #number of time steps
+```
+
+```{code-cell}
+---
+colab:
+  base_uri: https://localhost:8080/
+id: 0_30T42Xli1M
+outputId: f67ed77c-3c2e-4653-d7fa-57d47ffa7304
+---
+# Create qubits
+qubits = cirq.LineQubit.range(L)
+
+# Create a circuit
+test_circuit = cirq.Circuit()
+
+# define the local observable Z_2
+obs = Observable(PauliString("IIZ"))
+
+# we create different lists variables for average value of Z_2 and its mitigated quantities
+unmitigated_measurement=[1]
+exact_measurement=[1]
+mitigated_measurement_cdr=[1]
+mitigated_measurement_vncdr=[1]
+mitigated_measurement_zne=[1]
+
+# repeat the trotter evolution n_dt times and compute compute the unmitigated, exact and mitigated quantities for each step using Mitiq
+for ii in range(n_dt):
+  test_circuit+=trotter_evolution_H(L,h_z,h_x,Jdt)
+  unmitigated_measurement.append(obs.expectation(test_circuit, noisy_simulator).real)
+  exact_measurement.append(obs.expectation(test_circuit,exact_simulator))
+  mitigated_measurement_vncdr.append(cdr.execute_with_cdr(
+    test_circuit,
+    noisy_simulator,
+    observable=obs,
+    simulator=exact_simulator,
+    scale_factors=(1,3)
+).real)
+  mitigated_measurement_cdr.append(cdr.execute_with_cdr(
+    test_circuit,
+    noisy_simulator,
+    observable=obs,
+    simulator=exact_simulator,
+).real)
+  mitigated_measurement_zne.append(zne.execute_with_zne(
+    test_circuit,
+    noisy_simulator,
+    observable=obs,
+).real)
+
+  # this is to keep track of how fast the simulation is
+  print(ii)
+
+
+
+
+```
+
++++ {"id": "xXoS2SuzDCqX"}
+
+We plot all the exact, unmitigated, and different mitigated measurements to compare the results. As suggested in the original paper https://arxiv.org/pdf/2103.12680.pdf , the evolution of the $<Z_2(t)>$ is as follows
+
+$
+<σ^z_i(t)>=a_1 e^{-a_2 t}cos(a_3t)+a_4t+a_5
+$
+
+we used the trotterized simulation results to estimate parametrs $a_1$,...,$a_5$.
+
+```{code-cell}
+---
+colab:
+  base_uri: https://localhost:8080/
+  height: 450
+id: Y9xQMpwMOhI1
+outputId: 62ac6547-c275-4007-d62e-e410de58a817
+---
+import numpy
+from scipy.optimize import curve_fit
+import matplotlib.pyplot as plt
+
+# Define the average value function
+def sigma_z_t(x, a_1, a_2, a_3, a_4, a_5):
+    return a_1 * numpy.exp(-a_2 * x) * numpy.cos(a_3 * x) + a_4 * x + a_5
+
+# Generate some sample data
+x_data = np.linspace(0, n_dt*Jdt, n_dt+1)
+
+# Fit the cosine function to the data
+popt, pcov = curve_fit(sigma_z_t, x_data, exact_measurement)
+
+# Extract the optimized parameters
+a_1_opt, a_2_opt, a_3_opt, a_4_opt, a_5_opt = popt
+
+# Generate the fitted curve
+x_fit = np.linspace(0, n_dt*Jdt, 300)
+y_fit = sigma_z_t(x_fit, *popt)
```

Comment:
do you mean to do exact trotterized results? since I do not know how to simulate that Hamiltonian other than trattorization. I used two different names "exact" and "trotterized", I should use the same name for them, something like "exact trotterization".

let me know if this is what you are suggesting.